### ISSUE TITLE
Did L_var term in discriminative loss forget to divide the batch_size?

### ISSUE THREAD

**Issue by kyuusaku** [pqhieu/jsis3d#19]
https://github.com/pqhieu/jsis3d/blob/d4c27e6492e70c0e9c07e04c0bc348348892c674/losses/discriminative.py#L61

```
      for i in range(batch_size):
            n = size[i]
            loss += torch.sum(var[i,:,:n]) / torch.sum(masks[i,:,:n])
      loss /= batch_size
```

**Comment by pqhieu** [comment]
Hi @kyuusaku,

Good catch, let me correct the code. I'm not sure it will have any impact on the performance though.
